cg/meta/orders/rnafusion_submitter.py

```diff
@@ -1,5 +1,19 @@
+from typing import List
+
+from cg.exc import OrderError
 from cg.meta.orders.case_submitter import CaseSubmitter
+from cg.models.orders.order import OrderIn
+from cg.models.orders.samples import Of1508Sample
 
 
 class RnafusionSubmitter(CaseSubmitter):
-    pass
+    def validate_order(self, order: OrderIn) -> None:
+        """Validates that the order is correct."""
+        CaseSubmitter.validate_order(self=self, order=order)
```

Comment:
Is it common to pass self as a parameter? Seems a little confusing

Reply:
I think a more common way of doing method overriding in Python is to use the `super` method likes this
```python
super().validate_order(order=order)
```

I don't think it is necessary to pass self as an argument then

Reply:
I debated myself over which one to use. I liked this one because it looks clearer to me than `super()`. Maybe we can get some intuitive input from the others

Reply:
Changed to `super().validate_order(order=order)`